packages/thumbprint-react/components/Tooltip/index.tsx

```diff
@@ -219,7 +280,37 @@ export default function Tooltip({
                                 {/* We need to let the popper instance know when the contents of the tooltip change,
                                 so it can reposition itself.
                                 https://github.com/thumbtack/thumbprint-archive/issues/1033 */}
-                                <WhenChildrenChange do={scheduleUpdate}>{text}</WhenChildrenChange>
+                                <WhenChildrenChange do={scheduleUpdate}>
+                                    <div
+                                        onFocus={(): void => {
+                                            if (closeTimeout) {
+                                                window.clearTimeout(closeTimeout);
+                                            }
+                                        }}
+                                        onBlur={hide}
+                                        className={styles.tooltipContent}
+                                    >
+                                        {text}
+                                        {cta && cta.type === 'link' && (
+                                            <a
+                                                href={cta.href}
+                                                onClick={(): void => {
+                                                    if (cta.onClick) {
+                                                        cta.onClick();
+                                                    }
+                                                }}
+                                            >
+                                                {cta.text}
+                                            </a>
+                                        )}
+
+                                        {cta && cta.type === 'button' && (
+                                            <TextButton onClick={cta.onClick}>
+                                                {cta.text}
+                                            </TextButton>
+                                        )}
```

Comment:
Take a look at @egoens's latest comment. We should wrap in a `div` so that it becomes `display: block` rather tha appearing inline. Or wrap the `text` in a `p`.

Reply:
Done! Wrapped in Div.